ui/brave_custom_notification/public/cpp/constants.h

```diff
@@ -0,0 +1,152 @@
+const int kNotificationPreferredImageWidth = 360;
+const int kNotificationPreferredImageHeight = 240;
+const int kSmallImageSize = 16;
+const int kSmallImageSizeMD = 18;
+const int kSmallImagePadding = 4;
+
+// Limits.
+const size_t kMaxVisibleMessageCenterNotifications = 100;
+const size_t kMaxVisiblePopupNotifications = 3;
+
+// DIP dimension; H size of the whole card.
+const int kNotificationWidth = 360;
+
+// Colors.
+constexpr SkColor kMessageCenterBorderColor = SkColorSetRGB(0xC7, 0xCA, 0xCE);
+constexpr SkColor kMessageCenterShadowColor =
+    SkColorSetA(SK_ColorBLACK, 0.5 * 255);
+
+// Within a notification ///////////////////////////////////////////////////////
+
+// DIP dimensions (H = horizontal, V = vertical).
+
+const int kIconToTextPadding = 16;  // H space between icon & title/message.
+const int kTextTopPadding = 12;     // V space between text elements.
+const int kIconBottomPadding = 16;  // Minimum non-zero V space between icon
+                                    // and frame.
+// H space between the context message and the end of the card.
+const int kTextRightPadding = 23;
+const int kTextLeftPadding = kNotificationIconSize + kIconToTextPadding;
+const int kContextMessageViewWidth =
+    kNotificationWidth - kTextLeftPadding - kTextRightPadding;
+// space between buttons and frame.
+const int kControlButtonPadding = 2;
+const int kControlButtonBorderSize = 6;
+
+// Text sizes.
+const int kTitleFontSize = 14;        // For title only.
+const int kEmptyCenterFontSize = 13;  // For empty message only.
+const int kTitleLineHeight = 20;      // In DIPs.
+const int kMessageFontSize = 12;      // For everything but title.
+const int kMessageLineHeight = 18;    // In DIPs.
+
+// Colors.
+// Background of the card.
+constexpr SkColor kNotificationBackgroundColor = SK_ColorLTGRAY;
+// Background of the image.
+constexpr SkColor kImageBackgroundColor = kNotificationBackgroundColor;
+// Title, message, ...
+constexpr SkColor kRegularTextColorMD = SkColorSetRGB(0x21, 0x21, 0x21);
+constexpr SkColor kDimTextColorMD = SkColorSetRGB(0x75, 0x75, 0x75);
+// The focus border.
+constexpr SkColor kFocusBorderColor = SK_ColorTRANSPARENT;
+// Foreground of small icon image.
+constexpr SkColor kSmallImageMaskForegroundColor = SK_ColorWHITE;
+// Background of small icon image.
+constexpr SkColor kSmallImageMaskBackgroundColor =
+    SkColorSetRGB(0xa3, 0xa3, 0xa3);
+// Background of the close button and the settings button.
+#if defined(OS_CHROMEOS)
+constexpr SkColor kControlButtonBackgroundColor =
+    SkColorSetA(SK_ColorWHITE, 0.9 * 0xff);
+#else
+constexpr SkColor kControlButtonBackgroundColor = SK_ColorTRANSPARENT;
+#endif
+
+// Default accent color of notifications that are not generated by system.
+constexpr SkColor kNotificationDefaultAccentColor = gfx::kChromeIconGrey;
+
+// For list notifications.
+// Not used when --enabled-new-style-notification is set.
+const size_t kNotificationMaximumItems = 5;
```

Comment:
Is this going to be used? If not, can we please remove and check any other constants we no longer need for custom Brave Ads. Thanks